src/test/java/org/ethereum/beacon/discovery/message/handler/EnrUpdateTrackerTest.java

```diff
@@ -63,7 +63,7 @@ void shouldNotRequestPeersEnrWhenPongSequenceNumberIsLower() {
     final NodeRecord currentNodeRecord =
         TestUtil.generateNode(8000)
             .getNodeRecord()
-            .withUpdatedCustomField("test", Bytes.EMPTY, Bytes.EMPTY);
+            .withUpdatedCustomField("test", Bytes.EMPTY, Bytes.of(1));
```

Comment:
Just out of curiosity what was the reason of this change?

Reply:
The bouncy castle update performs additional verification on public keys and it won't accept an empty byte array anymore.